src/app/certificate/certificate.component.scss

```diff
@@ -0,0 +1,67 @@
+@import "../styles/main";
+
+#body {
+  text-align: center;
+  border: $red 2px;
```

Comment:
isn't there a variable for 2 px ?

Reply:
Currently, there is no such variable. A new variable for the single use case?

Reply:
It is a choice you can make, at least define the variable in this file instead of those common style files. It is fine even this way. But I thought 2px was being used else where also, as it is pretty common hence this comment.